tests/jerry/object-defineproperty.js

```diff
@@ -55,3 +55,17 @@ Object.defineProperty(obj, "prop", {
 
 var desc2 = Object.getOwnPropertyDescriptor(obj, "prop");
 assert (desc1.set === setter && desc2.set === undefined);
+
+obj = {}
+
+/* This error is thrown even in non-strict mode. */
+Object.defineProperty(obj, 'f', {
+  set: function(value) { throw 234 },
```

Comment:
Add `;` after 234. Not mandated by the standard but it is good practice not to excercise the semicolon insertion logic of the parser if it's not the feature we want to test.